### PR TITLE
:bug:Source Braze: Fix CanvasAnalytics Stream Null Data for step_stats, variant_stats

### LINKED ISSUE #30372: Source Braze: stream `canvas_analytics` no data for step_stats nor variant_stats
### Connector Name

source-braze

### Connector Version

[   {     "streamDescriptor": {       "name": "segments_analytics"     },     "streamState": {       "time": "2023-09-11",       "segment_id": "9356b165-8ad6-4148-9709-40f9e86a8907"     }   } ]

### What step the error happened?

Other

### Revelant information

braze canvas analytics only have entries, revenue and conversion data, but no opens, clicks, sends, unsub, spam, etc. this braze api endpoint https://www.braze.com/docs/api/endpoints/export/canvas/get_canvas_analytics/ is the endpoint airbyte uses to extract canvas_analytics stream for the braze connector. for some reason, airbyte does NOT include STEP_STATS and VARIANT_STATS data, which is why on Mode/snowflake we get null values for those fields. 

The fix should be easy here, you need to add the following parameters: 
- include_step_breakdown: true
- include_variant_breakdown: true

I tried this fix manually and it did give me the step_stats and variant_stats for a campaign id data that i was looking for to get additional data points from canvases. The endpoint above should make it pretty clear. 

### Relevant log output

_No response_

### Contribute

- [X] Yes, I want to contribute

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Fixes #30372

## How
*Describe the solution*

## Recommended reading order
1. `manifest.yaml`
2. `canvases_analytics.json`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-braze/source_braze/manifest.yaml [hunk 1]
```diff
@@ -164,6 +164,11 @@ definitions:
       $ref: "#/definitions/datetime_14d_incremental_sync"
     retriever:
       $ref: "#/definitions/non_paginated_retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          include_variant_breakdown: true
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-braze/source_braze/manifest.yaml:170]: Are additional fields declared in the schema?

**Review by lazebnyi — COMMENTED** [review]
need bump version and change log in docs

File: airbyte-integrations/connectors/source-braze/source_braze/manifest.yaml (hunk 1 above)
Comment (topefolorunso) [reply airbyte-integrations/connectors/source-braze/source_braze/manifest.yaml:170]: Yes, additional properties are declared here - https://github.com/airbytehq/airbyte/pull/31607/files#diff-d8a79b4cd6e3c9444fafbbe6071a31d83a364e018bb4782e6e3cf07722862778R4

**Comment by topefolorunso** [comment]
> need bump version and change log in docs

Done @lazebnyi

**Review by marcosmarxm — APPROVED** [review]
Thanks @topefolorunso confirmed its working now
image

**Pull request merged**